crates/uv/tests/it/workspace_dir.rs

```diff
@@ -0,0 +1,134 @@
+use std::env;
+use std::path::PathBuf;
+
+use anyhow::Result;
+use assert_cmd::assert::OutputAssertExt;
+use assert_fs::fixture::PathChild;
+
+use crate::common::{TestContext, copy_dir_ignore, uv_snapshot};
+
+fn workspaces_dir() -> PathBuf {
+    env::current_dir()
```

Comment:
We usually read the manifest directory

https://github.com/astral-sh/uv/blob/8b479efd2fd03d5f92ce68e9cd98cf27a23ebc82/crates/uv/tests/it/common/mod.rs#L621-L628

I'd move that part out into a helper in `it/common/mod.rs`